atom/common/api/resources/windows_bindings.js

```diff
@@ -17,7 +17,21 @@ binding.registerCustomHook(function (bindingsAPI, extensionId) {
   })
 
   apiFunctions.setHandleRequest('get', function (windowId, getInfo, cb) {
-    console.warn('chrome.windows.get is not supported yet')
+    if (arguments.length === 1) {
+      cb = arguments[0]
+    } else if (arguments.length == 2) {
+      getInfo = arguments[0]
+      cb = arguments[1]
+    } else {
+      windowId = arguments[0]
+      getInfo = arguments[1]
+      cb = arguments[2]
+    }
+    var responseId = ++id
```

Comment:
we've moved away from using `++id` because it can produce duplicate requests across multiple windows. See `tabs_bindings.js` for usage of `ipc.guid()`